internal/artieclient/ssh_tunnel.go

```diff
@@ -34,12 +38,12 @@ func (sc SSHTunnelClient) Get(ctx context.Context, sshTunnelUUID string) (SSHTun
 	return makeRequest[SSHTunnel](ctx, sc.client, http.MethodGet, path, nil)
 }
 
-func (sc SSHTunnelClient) Create(ctx context.Context, name, host, username string, port int32) (SSHTunnel, error) {
+func (sc SSHTunnelClient) Create(ctx context.Context, sshTunnel BaseSSHTunnel) (SSHTunnel, error) {
 	body := map[string]any{
-		"name":     name,
-		"host":     host,
-		"port":     port,
-		"username": username,
+		"name":     sshTunnel.Name,
+		"host":     sshTunnel.Host,
+		"port":     sshTunnel.Port,
+		"username": sshTunnel.Username,
```

Comment:
Could you just pass `sshTunnel` directly to `makeRequest`?

Reply:
ooh good call